ios/Classes/SwiftAirshipPlugin.swift

```diff
@@ -196,3 +248,29 @@ UADeepLinkDelegate, UAPushNotificationDelegate, UAInboxDelegate {
         }
     }
 }
+
+private extension UACustomEvent {
+    func parseProperties(properties:Dictionary<String, Any>) {
```

Comment:
You can do extensions on kotlin as well if you want the code to look the same

Reply:
added